1.8/usage/cli/install.md

```diff
@@ -79,20 +80,91 @@ The easiest way to install the DC/OS CLI is by clicking the signpost icon in the
     Get detailed command description with 'dcos <command> --help'.
     ```
 
+# <a name="osx"></a>Installing on OS X
+
+The easiest way to install the DC/OS CLI is by clicking the signpost icon in the lower left corner of the DC/OS web interface. Or, you can manually install the CLI by following these instructions.
+
+#### Prerequisites
+
+*   A system external to your DC/OS cluster that you can install the CLI.
+*   Network access from the external system to your DC/OS cluster.
+*   A command-line environment, such as Terminal.
+*   cURL: Installed by default on OS X
+
+#### Installing the DC/OS CLI
+
+1.  Download the DC/OS CLI binary (`dcos`) to your local directory (for example, `/usr/local/bin/`).
+
+    ```bash
+    $ curl https://downloads.dcos.io/binaries/cli/darwin/x86-64/0.4.10/dcos
+    ```
+    
+    **Important:** The CLI must be installed on a system that is external to your DC/OS cluster.
+
+1.  Make the CLI binary executable.
+ 
+    ```bash
+    chmod +x <path/to/download/directory>/dcos
+    ```
+
+1.  Point the CLI to your DC/OS master node. In this example, `http://example.com` is the master node IP address.
+
+    ```bash
+    dcos config set core.dcos_url http://example.com
+    ```
+
+    You are now ready to use the CLI. Enter `dcos` to get started.
+
+1.  Authenticate your CLI with your master node and set the auth token:
+   
+    ```bash
+    $ dcos auth login
+    ```
+    
+    You will be prompted for your username and password. For more information on the `superuser` and security, see the [documentation](/1.7/administration/id-and-access-mgt/).
```

Comment:
This is only true for enterprise. For open, users will be prompted with link to enter a token. I think we should remove this part from the install instructions since users will automatically be prompted with authentication when it's needed. @joel-hamill what do you think?